internal/pkg/server/agent_integration_test.go

```diff
@@ -177,6 +189,7 @@ func TestAgent(t *testing.T) {
 	})
 	require.NoError(t, err)
 	expected = makeExpected(agentID, 1, inputSource, 3, goodSource)
+	expected.Units[0].LogLevel = proto.UnitLogLevel_DEBUG
```

Comment:
I'm not sure how the v2 mocker works here. When I tried to add create a new `expected` after it went healthy the log level was ignored